bokeh/plot_object.py

```diff
@@ -227,18 +229,12 @@ def references(self):
         """Returns all ``PlotObjects`` that this object has references to. """
         return set(self.collect_plot_objects(self))
 
-    #---------------------------------------------------------------------
-    # View Model connection methods
-    #
-    # Whereas a rich client rendering framework can maintain view state
-    # alongside model state, we need an explicit send/receive protocol for
-    # communicating with a set of view models that reside on the front end.
-    # Many of the calls one would expect in a rich client map instead to
-    # batched updates on the M-VM-V approach.
-    #---------------------------------------------------------------------
     def vm_props(self, changed_only=True):
         """ Returns the ViewModel-related properties of this object.
 
+            In the future this method may always return all properties,
+            ignoring the changed_only argument.
```

Comment:
The indent here is not needed, and may be misinterpreted/mis-renderered by Sphinx. Tho, I would also put this in a `..note::` admonition.